internal/pkg/cli/deploy/deploy.go

```diff
@@ -499,6 +507,91 @@ func (d *workloadDeployer) UploadArtifacts() (*UploadArtifactsOutput, error) {
 	}, nil
 }
 
+// UploadArtifacts uploads the deployment artifacts such as the container image, custom resources, addons and env files.
+func (d *lbWebSvcDeployer) UploadArtifacts() (*UploadArtifactsOutput, error) {
+	out, err := d.workloadDeployer.UploadArtifacts()
+	if err != nil {
+		return nil, err
+	}
+	if !d.uploadCustomResourceFlag {
+		return out, nil
+	}
+	urls, err := d.uploadCustomResources(manifest.LoadBalancedWebServiceType, customresource.LBWS)
+	if err != nil {
+		return nil, err
+	}
+	out.CustomResourceURLs = urls
```

Comment:
what if these were in the `workloadDeployer.UploadArtifacts()` func instead? is there a reason for splitting it by workload type? they just seem pretty similar now
L502:
```
if !d.uplaodCustomResourceFlag {
    return &UploadArtifcatsOutput{...}
}

switch d.mft.(type) {
case lbws:
    out.CustomResourceURLs, err = d.uploadCustomResources(lbwsType, lbws)
// etc
}
if err != nil {
}

return out
```